src/features/simd.md

```diff
@@ -0,0 +1,179 @@
+
+The WebAssembly SIMD proposal defines a portable, performant subset of SIMD operations that are available across most modern architectures. This proposal derived many elements from the [SIMD.js proposal](https://github.com/tc39/ecmascript_simd), which in turn was originally derived from the [Dart SIMD](https://www.researchgate.net/publication/261959129_A_SIMD_programming_model_for_dart_javascriptand_other_dynamically_typed_scripting_languages) specification. The SIMD.js proposal was an API proposed at TC39 with new types and functions for performing SIMD computations, but this was archived in favor of supporting SIMD operations more transparently in WebAssembly. The [WebAssembly SIMD proposal](https://github.com/WebAssembly/simd) was introduced as away for browsers to take advantage of the data level parallelism using the underlying hardware.
+
+## WebAssembly SIMD Proposal
+
+The high level goal of the WebAssembly SIMD proposal is to introduce vector operations to the WebAssembly Specification, in a way that guarantees portable performance. 
+
+The set of SIMD instructions is large, and varied across architectures. The set of operations included in the WebAssembly SIMD proposal consist of operations that are well supported on a wide variety of platforms, and are proven to be performant. To this end, the current proposal is limited to standardizing Fixed-Width 128-bit SIMD operations. 
+
+The current proposal introduces a new v128 value type, and a number of new operations that operate on this type. The criteria used to determine these operations are -
+
+* The operations should be well supported across multiple modern architectures.
+
+* Performance wins should be positive across multiple relevant arches within an instruction group.
+
+* The chosen set of operations should minimize performance cliffs if any.
+
+ 
+
+The proposal is in active development, both V8, and the toolchain have working prototype implementations for experimentation. As these are prototype implementations, they are subject to change as new operations are added to the proposal.
+
+## Using WebAssembly SIMD
+
+### Enabling experimental SIMD support in Chrome
+
+WebAssembly SIMD support is prototyped behind a flag in chrome, to try out the SIMD support on the browser, pass --enable-features=WebAssemblySimd, or toggle the WebAssemblySimd flag in chrome://flags. This work is bleeding edge, and continuously being worked on. To minimize the chances of breakage, please use the latest version of the toolchain as detailed below, and a recent Chrome Canary. If something doesn’t look right, please file a bug at crbug.com/v8.
```

Comment:
"crbug.com/v8" should be a link (or does that happen automatically?)

Reply:
Probably not without an explicit `https://`, need to check.

Reply:
In general, we don't do auto-linking on v8.dev. Do one of these:

- To turn `https://example.com/` into a link with custom link text, use `[foo bar baz](https://example.com/)`
- To turn `https://example.com/` into a link, wrap it in `<>`: `<https://example.com/>`

In 99% of cases you want the former.

Reply:
This should be fixed now.